mdx-models/src/main/java/com/mx/path/model/mdx/model/profile/Profile.java

```diff
@@ -17,56 +23,13 @@ public final class Profile extends MdxBase<Profile> {
   private String middleName;
   private String ssn;
 
-  public Profile() {
-  }
-
-  public LocalDate getBirthDate() {
-    return birthDate;
-  }
-
-  public void setBirthDate(LocalDate birthDate) {
-    this.birthDate = birthDate;
-  }
-
-  public String getFirstName() {
-    return firstName;
-  }
-
-  public void setFirstName(String firstName) {
-    this.firstName = firstName;
-  }
-
-  public Gender getGender() {
-    return gender;
-  }
-
-  public void setGender(Gender gender) {
-    this.gender = gender;
-  }
-
-  public String getLastName() {
-    return lastName;
-  }
-
-  public void setLastName(String lastName) {
-    this.lastName = lastName;
-  }
-
-  public String getMiddleName() {
-    return middleName;
-  }
-
-  public void setMiddleName(String middleName) {
-    this.middleName = middleName;
-  }
-
-  public String getSsn() {
-    return ssn;
-  }
-
-  public void setSsn(String ssn) {
-    this.ssn = ssn;
-  }
+  // internal fields
+  @Internal
+  @SerializedName("remote_deposit_eligible")
+  private Boolean remoteDepositEligible;
+  @Internal
+  @SerializedName("remote_deposit_enrolled")
+  private Boolean remoteDepositEnrolled;
```

Comment:
`@SerializedName` is not necessary since MDX models are already serialized using `FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES` by default. We would only need to use the annotation if we want to create a non-default serialized name.

Reply:
Thanks! I always forget about that. Fixed